src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -406,7 +441,7 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
 
             if (TryGetBooleanQueryParameterValue(queryNameValuePairs, FetchInputParameter, out bool fetchInput))

```

Comment:
This parameter name should be `fetchState` and not `fetchInput`. We should also update the local variable to reflect this.